docs/functions/data.rst

```diff
@@ -99,6 +99,16 @@ Environment
    .. versionchanged:: 2.0
       Return method changed from stdout.
 
+.. function:: _lp_dirstack() -> var:lp_dirstack
+
+    Returns true if directory stack support is enabled and the directory stack
```

Comment:
Wrap true in tics:
```suggestion
    Returns ``true`` if directory stack support is enabled and the directory stack
```

Reply:
This will make this line longer than 80 characters, so you'll need to reformat the whole block.